### PR TITLE
Budgets indicators page

### LINKED ISSUE #1510: Budgets / Indicators page
- New subsection in Budgets module. A simple page that shows a grid with cards as in /observatorio
- In the submenu, at the end: "Indicadores"
- The indicators: the ones we have already calculated: https://github.com/PopulateTools/populate-data-indicators/tree/master/i-presupuestos-municipales/datasets

ds-ahorro-bruto 	
ds-ahorro-neto 	
ds-autonomia-fiscal 	
ds-capacidad-autofinanciacion 	
ds-carga-financiera 	
ds-coste-deuda 	
ds-financiacion-inversiones 	
ds-indice-ahorro-bruto 	
ds-indice-ahorro-neto 	
ds-inversion-por-habitante
ds-nivel-endeudamiento 	
ds-presion-fiscal-por-habitante 	
ds-presupuestos-municipales-partida
ds-presupuestos-municipales-total
ds-riesgo-financiero
ds-rigidez-gasto 	

- We'll use the basic card (the first one you can see in https://portalobert.esplugues.cat/observatorio): 
  imagen

Related: https://github.com/PopulateTools/issues/issues/267

### PR DESCRIPTION
**Pull request by apradillap**
Closes #1510 

## :v: What does this PR do?

New subsection in Budgets module, Indicators:

- ds-ahorro-bruto
- ds-ahorro-neto
- ds-autonomia-fiscal
- ds-capacidad-autofinanciacion
- ds-carga-financiera
- ds-coste-deuda
- ds-financiacion-inversiones
- ds-indice-ahorro-bruto
- ds-indice-ahorro-neto
- ds-inversion-por-habitante
- ds-nivel-endeudamiento
- ds-presion-fiscal-por-habitante
- ds-presupuestos-municipales-partida
- ds-presupuestos-municipales-total
- ds-riesgo-financiero
- ds-rigidez-gasto

## :mag: How should this be manually tested?

Enable the section:

/admin/gobierto_budgets/options

See the section:

/presupuestos/indicadores

### Before this PR

Nothing, new section

### After this PR

captura de pantalla 2018-03-14 a las 12 46 49

captura de pantalla 2018-03-14 a las 12 47 14

## :shipit: Does this PR changes any configuration file?

No

## Pending

- [ ] As we have brought the styles of another module GobiertoObservatory, having "body with the classes governing_budgets indicators index indicators_index" are not shown properly

### CODE REVIEW

**Comment by apradillap** [comment]
Gross savings amounts do not match:

Blat: Ahorro bruto 2016 - 1.662.385.340 €
Indicadores: Ahorro bruto 2016 - 1.436.346.318 €
Fórmula: Ahorro bruto 2016 - 745.573.691 €

total_income = 0
(1..5).each do |code|
  total_income += get_income_budget_line(entity_id, year, code)
end

total_expense = 0
(1..4).each do |code|
  total_expense += get_expense_budget_line(entity_id, year, code)
end

value = (total_income - total_expense).round(2)

total_income = 2.246.767.421 + 138.541.479 + 571.790.129 + 1.415.869.649 + 92.208.959
total_expense = 716.297.909 + 2.075.959.271 + 519.532.188 + 407.814.578
value = 745.573.691 €

File: app/views/gobierto_budgets/indicators/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,53 @@
+<script type="text/javascript">
+  window.populateDataYear = {
```
Comment (ferblape) [inline app/views/gobierto_budgets/indicators/index.html.erb]: I think this is the problem of the JS error. You already have a year here: https://github.com/PopulateTools/gobierto/blob/master/app/views/layouts/_head.html.erb#L49, so you don't need to define this.

**Review by ferblape — CHANGES_REQUESTED** [review]

**Comment by furilo** [comment]
Looking at Madrid data: 

- [x] we show the option to select 2018, but we don't have budget data for that year
- [x] 2016 is showing the same data than 2017. I guess its because we don't have execution data for 2017. So if we can't calculate them, we shouldn't show the page for 2017

**Comment by furilo** [comment]
- [x] In Getafe there are some indicators with no value (for Madrid also, I think). If we are not able to calculate them, please remove it from the page.

File: app/views/gobierto_budgets/indicators/index.html.erb (hunk 1 above)
Comment (apradillap) [reply app/views/gobierto_budgets/indicators/index.html.erb]: @ferblape, The first thing I did as you rightly say was use window.populateData.year, but If I use the variable window.populateData.year it does not filter me, it is fixed, it is always 2015, and I need it to take the different years that I change with the year selector.

Comment (ferblape) [reply app/views/gobierto_budgets/indicators/index.html.erb]: Ok, could check the JS error in the budgets home page?

Comment (apradillap) [reply app/views/gobierto_budgets/indicators/index.html.erb]: I've fixed it. Other javascript errors appear, which are also in production.

**Comment by apradillap** [comment]
Javascript errors fixed.

**Comment by ferblape** [comment]
I have added the units (euros, percentages) to the cards and fixed a JS error:

screen shot 2018-03-23 at 17 40 19

**Review by ferblape — APPROVED** [review]

**Pull request merged**